fakestorage/upload.go

```diff
@@ -411,7 +404,7 @@ func (s *Server) resumableUpload(bucketName string, r *http.Request) jsonRespons
 
 // uploadFileContent accepts a chunk of a resumable upload
 //
-// A resumable upload is sent in one or more chunks. The request's
+// A resumeable upload is sent in one or more chunks. The request's
```

Comment:
```suggestion
// A resumable upload is sent in one or more chunks. The request's
```
Typo? [resumable](https://www.merriam-webster.com/dictionary/resumable) is correct